source/includes/_application-api.md

```diff
@@ -0,0 +1,453 @@
+
+
+Before sending any subscription, you need to _connect_ to the web socket to open the two-way communication channel. This connection
+is done differently depending on the client/library you use to manage web sockets. In this section, we will show examples 
+using the ["wscat"](https://github.com/websockets/wscat) command line program. 
+
+Once you have connected successfully, you can use this channel to:
+
+- Send the subscription messages
+- Listen for messages sent by the server with data corresponding to your active subscriptions. 
+
+The structure of the "subscription" (the body of the message) is exactly the same as the "query" operation:
+
+```graphql
+subscription {
+  read_model_name(id: "<id of the read model>") {
+    selection_field_list
+  }
+}
+```
+
+Where:
+
+- _read_model_name_ is the name of the class corresponding to the read model you want to subscribe to.
+- _&lt;id of the read model&gt;_ is the ID of the specific read model instance you are interested in.
+- _selection_field_list_ is a list with the names of the specific read model fields you want to get when data is sent back to you.
+
+In the following examples we use ["wscat"](https://github.com/websockets/wscat) to connect to the web socket and send a subscription
+to the read model `CartReadModel` with ID "demo"
+
+<aside class="notice">
+Remember that in case you want to subscribe to a read model that is restricted to a specific set of roles, you must send, 
+in the <em>connection operation</em>, the <strong>access token</strong> in the <strong>"Authorization"</strong> header: 
+<em>"Authorization: Bearer &lt;token retrieved upon sign-in&gt;"</em>
+</aside>
+
+> Connecting to the web socket:
+
+```sh
+ wscat -c <websocketURL>
+```
+
+> Sending a message with the subscription
+
+```json
+{"query": "subscription { CartReadModel(id:\"demo\") { id items } }" }
+```
+
+After a successful subscription, you won't receive anything in return. Now, every time the read model you subscribed to
+is modified, a new incoming message will appear in the socket with the updated version of the read model. This message
+will have exactly the same format as if you were done a query with the same parameters.
+
+Following the previous example, we now send a command (using a "mutation" operation) that adds
+a new item with sku "ABC_02" to the `CartReadModel`. After it has been added, we receive the updated version of the read model through the
+socket.
+
+> Send the Command
+
+```
+URL: "<httpURL>/graphql"
+```
+```graphql
+mutation {
+  ChangeCart(input: {
+    cartId: "demo"
+    sku: "ABC_02"
+    quantity: 3
+  })
+}
+```
+
+> The following message appears in the socket
+
+```json
+{
+  "data": {
+    "CartReadModel": {
+      "id": "demo",
+      "items": [
+        {
+          "sku": "ABC_01",
+          "quantity": 2
+        },
+        {
+          "sku": "ABC_02",
+          "quantity": 3
+        }
+      ]
+    }
+  }
+}
+```
```

Comment:
I think that it could be a good idea to show how to do this with ApolloClient in addition to the raw requests to show more realistic usage beyond pure documentation. We can do that in the new doc pages, no need to add it in this PR. /cc @NickSeagull @itrion

Reply:
Totally. I left that for the future because we still need to ensure Booster is compatible with the subscriptions-transport-ws to get full compatibility with Apollo

Reply:
BTW, feel free to merge the PR if you need that!